snap/snapcraft.yaml

```diff
@@ -28,8 +28,70 @@ slots:
   rt-tests-posix-mq:
     interface: posix-mq
     path:
-      - /syncmsg
-      - /testmsg
+      - /syncmsg0
+      - /syncmsg1
+      - /syncmsg2
+      - /syncmsg3
+      - /syncmsg4
+      - /syncmsg5
+      - /syncmsg6
+      - /syncmsg7
+      - /syncmsg8
+      - /syncmsg9
+      - /syncmsg10
+      - /syncmsg11
+      - /syncmsg12
+      - /syncmsg13
+      - /syncmsg14
+      - /syncmsg15
+      - /syncmsg16
+      - /syncmsg17
+      - /syncmsg18
+      - /syncmsg19
+      - /syncmsg20
+      - /syncmsg21
+      - /syncmsg22
+      - /syncmsg23
+      - /syncmsg24
+      - /syncmsg25
+      - /syncmsg26
+      - /syncmsg27
+      - /syncmsg28
+      - /syncmsg29
+      - /syncmsg30
+      - /syncmsg31
+      - /testmsg0
+      - /testmsg1
+      - /testmsg2
+      - /testmsg3
+      - /testmsg4
+      - /testmsg5
+      - /testmsg6
+      - /testmsg7
+      - /testmsg8
+      - /testmsg9
+      - /testmsg10
+      - /testmsg11
+      - /testmsg12
+      - /testmsg13
+      - /testmsg14
+      - /testmsg15
+      - /testmsg16
+      - /testmsg17
+      - /testmsg18
+      - /testmsg19
+      - /testmsg20
+      - /testmsg21
+      - /testmsg22
+      - /testmsg23
+      - /testmsg24
+      - /testmsg25
+      - /testmsg26
+      - /testmsg27
+      - /testmsg28
+      - /testmsg29
+      - /testmsg30
+      - /testmsg31
```

Comment:
This is getting out of hand. I suggest making this work only with the default number of threads, i.e. 1 thread when `--threads` is not set. The packaging can't make assumption about how many threads may be set for a test or in case `--threads` is set without a value, how many CPUs are available. Allowing the default and requiring dev mode for everything else is simpler to explain in the doc and understand.

For reference:
```
$ rt-tests.pmqtest --help
...
-t       --threads         one thread per available processor
-t [NUM] --threads=NUM     number of threads:
                           without NUM, threads = max_cpus
                           without -t default = 1
...

```